--- origsrc/plplot-5.10.0/bindings/ocaml/CMakeLists.txt	2014-04-07 17:11:46.653949300 -0500
+++ src/plplot-5.10.0/bindings/ocaml/CMakeLists.txt	2014-04-07 17:13:49.223013600 -0500
@@ -100,7 +100,7 @@ if(ENABLE_ocaml)
     ${CMAKE_CURRENT_BINARY_DIR}/libplplot_stubs.a
     COMMAND ${OCAMLC} -ccopt -I${CAMLIDL_LIB_DIR} -c ${CMAKE_CURRENT_BINARY_DIR}/plplot_core_stubs.c
     COMMAND ${OCAMLC} -ccopt -I${CMAKE_SOURCE_DIR}/include -ccopt -I${CMAKE_BINARY_DIR}/include -ccopt -I${CMAKE_SOURCE_DIR}/lib/qsastime -ccopt -I${CMAKE_BINARY_DIR} -ccopt -I${CAMLIDL_LIB_DIR} -ccopt -DHAVE_CONFIG_H -c ${CMAKE_CURRENT_SOURCE_DIR}/plplot_impl.c
-    COMMAND ${OCAMLMKLIB} -o plplot_stubs -L${CAMLIDL_LIB_DIR} -lcamlidl -L${CMAKE_BINARY_DIR}/src -lplplot${LIB_TAG} ${CMAKE_CURRENT_BINARY_DIR}/plplot_core_stubs.o ${CMAKE_CURRENT_BINARY_DIR}/plplot_impl.o ${ocaml_STATIC_FLAGS}
+    COMMAND ${OCAMLMKLIB} -o plplot_stubs -L${CAMLIDL_LIB_DIR} -lcamlidl -L${CMAKE_BINARY_DIR}/dll -lplplot${LIB_TAG} ${CMAKE_CURRENT_BINARY_DIR}/plplot_core_stubs.o ${CMAKE_CURRENT_BINARY_DIR}/plplot_impl.o ${ocaml_STATIC_FLAGS}
     DEPENDS
     ${CMAKE_CURRENT_BINARY_DIR}/plplot_core_stubs.c
     ${CMAKE_CURRENT_SOURCE_DIR}/plplot_impl.c
@@ -159,7 +159,7 @@ if(ENABLE_ocaml)
   add_custom_command(
     OUTPUT
     ${CMAKE_CURRENT_BINARY_DIR}/plplot.cma
-    COMMAND ${OCAMLC} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plplot.cma ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmo ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmo -dllib -lplplot_stubs -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplplot_stubs -ccopt -L${CAMLIDL_LIB_DIR} -cclib -lcamlidl -ccopt -L${CMAKE_BINARY_DIR}/src -cclib -lplplot${LIB_TAG} -dllpath ${CMAKE_BINARY_DIR}/src ${ocaml_STATIC_FLAGS}
+    COMMAND ${OCAMLC} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plplot.cma ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmo ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmo -dllib -lplplot_stubs -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplplot_stubs -ccopt -L${CAMLIDL_LIB_DIR} -cclib -lcamlidl -ccopt -L${CMAKE_BINARY_DIR}/dll -cclib -lplplot${LIB_TAG} -dllpath ${CMAKE_BINARY_DIR}/dll ${ocaml_STATIC_FLAGS}
     DEPENDS
     ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmo
     ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmo
@@ -212,7 +212,7 @@ if(ENABLE_ocaml)
       OUTPUT
       ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmxa
       ${CMAKE_CURRENT_BINARY_DIR}/plplot.a
-      COMMAND ${OCAMLOPT} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmxa ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmx ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmx -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplplot_stubs -ccopt -L${CAMLIDL_LIB_DIR} -cclib -lcamlidl -ccopt -L${CMAKE_BINARY_DIR}/src -cclib -lplplot${LIB_TAG} ${ocaml_STATIC_FLAGS}
+      COMMAND ${OCAMLOPT} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmxa ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmx ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmx -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplplot_stubs -ccopt -L${CAMLIDL_LIB_DIR} -cclib -lcamlidl -ccopt -L${CMAKE_BINARY_DIR}/dll -cclib -lplplot${LIB_TAG} ${ocaml_STATIC_FLAGS}
       DEPENDS
       ${CMAKE_CURRENT_BINARY_DIR}/plplot_core.cmx
       ${CMAKE_CURRENT_BINARY_DIR}/plplot.cmx
--- origsrc/plplot-5.10.0/bindings/ocaml/plcairo/CMakeLists.txt	2014-04-07 17:11:46.656949700 -0500
+++ src/plplot-5.10.0/bindings/ocaml/plcairo/CMakeLists.txt	2014-04-07 17:14:08.360943800 -0500
@@ -37,7 +37,7 @@ if(ENABLE_ocaml AND OCAML_HAS_CAIRO)
     ${CMAKE_CURRENT_BINARY_DIR}/dllplcairo_stubs.so
     ${CMAKE_CURRENT_BINARY_DIR}/libplcairo_stubs.a
     COMMAND ${OCAMLC} -ccopt "${CAIRO_COMPILE_FLAGS}" -cclib "${CAIRO_LINK_FLAGS}" -ccopt -I${CMAKE_SOURCE_DIR}/include -ccopt -I${CMAKE_BINARY_DIR}/include -ccopt -I${CMAKE_SOURCE_DIR}/lib/qsastime  -ccopt -I${CMAKE_BINARY_DIR} -ccopt -DHAVE_CONFIG_H -c ${CMAKE_CURRENT_SOURCE_DIR}/plcairo_impl.c
-    COMMAND ${OCAMLMKLIB} -o plcairo_stubs ${CAIRO_LINK_FLAGS_LIST} -L${CMAKE_BINARY_DIR}/src -lplplot${LIB_TAG} ${CMAKE_CURRENT_BINARY_DIR}/plcairo_impl.o
+    COMMAND ${OCAMLMKLIB} -o plcairo_stubs ${CAIRO_LINK_FLAGS_LIST} -L${CMAKE_BINARY_DIR}/dll -lplplot${LIB_TAG} ${CMAKE_CURRENT_BINARY_DIR}/plcairo_impl.o
     DEPENDS
     ${CMAKE_CURRENT_SOURCE_DIR}/plcairo_impl.c
     plplot${LIB_TAG}
@@ -88,7 +88,7 @@ if(ENABLE_ocaml AND OCAML_HAS_CAIRO)
   add_custom_command(
     OUTPUT
     ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cma
-    COMMAND ${OCAMLC} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cma -I ${PATH_TO_PLPLOT_CMA} plplot.cma ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmo -dllib -lplcairo_stubs -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplcairo_stubs -ccopt -L${CMAKE_BINARY_DIR}/src -cclib -lplplot${LIB_TAG} -dllpath ${CMAKE_BINARY_DIR}/src
+    COMMAND ${OCAMLC} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cma -I ${PATH_TO_PLPLOT_CMA} plplot.cma ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmo -dllib -lplcairo_stubs -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplcairo_stubs -ccopt -L${CMAKE_BINARY_DIR}/dll -cclib -lplplot${LIB_TAG} -dllpath ${CMAKE_BINARY_DIR}/dll
     DEPENDS
     ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmo
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
@@ -135,7 +135,7 @@ if(ENABLE_ocaml AND OCAML_HAS_CAIRO)
       OUTPUT
       ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmxa
       ${CMAKE_CURRENT_BINARY_DIR}/plcairo.a
-      COMMAND ${OCAMLOPT} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmxa -I ${PATH_TO_PLPLOT_CMXA} ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmx -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplcairo_stubs -ccopt -L${CMAKE_BINARY_DIR}/src -cclib -lplplot${LIB_TAG}
+      COMMAND ${OCAMLOPT} -a -o ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmxa -I ${PATH_TO_PLPLOT_CMXA} ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmx -ccopt -L${CMAKE_CURRENT_BINARY_DIR} -cclib -lplcairo_stubs -ccopt -L${CMAKE_BINARY_DIR}/dll -cclib -lplplot${LIB_TAG}
       DEPENDS
       ${CMAKE_CURRENT_BINARY_DIR}/plcairo.cmx
       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
--- origsrc/plplot-5.10.0/bindings/qt_gui/pyqt4/CMakeLists.txt	2013-12-11 03:35:08.000000000 -0600
+++ src/plplot-5.10.0/bindings/qt_gui/pyqt4/CMakeLists.txt	2014-04-07 17:12:19.572129400 -0500
@@ -69,16 +69,16 @@ if(ENABLE_pyqt4)
     ${plplot_pyqt4_SRC}
     )
 
-  if(WIN32_OR_CYGWIN)
+  if(WIN32_AND_NOT_CYGWIN)
     set_target_properties(
       plplot_pyqt4
       PROPERTIES
       PREFIX ""
       SUFFIX ".pyd"
       )
-  else(WIN32_OR_CYGWIN)
+  else(WIN32_AND_NOT_CYGWIN)
     set_target_properties(plplot_pyqt4 PROPERTIES PREFIX "")
-  endif(WIN32_OR_CYGWIN)
+  endif(WIN32_AND_NOT_CYGWIN)
 
   target_link_libraries(
     plplot_pyqt4
--- origsrc/plplot-5.9.9/drivers/CMakeLists.txt	2011-10-12 19:43:01.000000000 -0500
+++ src/plplot-5.9.9/drivers/CMakeLists.txt	2012-07-11 14:44:11.226978300 -0500
@@ -165,7 +165,6 @@ if(ENABLE_DYNDRIVERS)
 	${SOURCE_ROOT_NAME}
 	PROPERTIES
 	PREFIX ""
-	SUFFIX ${DYNAMIC_SUFFIX}
 	INSTALL_RPATH
 	"${${SOURCE_ROOT_NAME}_RPATH}"
 	)
@@ -174,7 +173,6 @@ if(ENABLE_DYNDRIVERS)
 	${SOURCE_ROOT_NAME}
 	PROPERTIES
 	PREFIX ""
-	SUFFIX ${DYNAMIC_SUFFIX}
 	)
     endif(USE_RPATH)
 
--- origsrc/plplot-5.9.9/drivers/test-drv-info.c	2011-10-12 19:43:01.000000000 -0500
+++ src/plplot-5.9.9/drivers/test-drv-info.c	2012-07-11 14:39:19.543153000 -0500
@@ -55,7 +55,7 @@ main( int argc, char* argv[] )
     signal( SIGSEGV, catch_segv );
 
     lt_dlinit();
-#if defined ( LTDL_WIN32 ) || defined ( __CYGWIN__ )
+#if defined ( LTDL_WIN32 )
     snprintf( drvspec, DRVSPEC_LEN, "%s", drvnam );
 #else
     snprintf( drvspec, DRVSPEC_LEN, "%s/%s", plGetDrvDir(), drvnam );
